collections/hub/sdk_scheme_adapter/bulk/basic/bulk-happy-path.json

```diff
@@ -5,10 +5,7 @@
       "id": 1,
       "name": "TC-BHP1. When there are 4 transfers in the bulk request 2 of which that have the same receiver fsp id and the others has a different receiver fsp id and acceptParty is true for all the transfers, Then the PUT response should have details of all the 4 transfers.",
       "fileInfo": {
-        "path": "hub/sdk_scheme_adapter/bulk/basic/bulk-happy-path.json",
-        "labels": [
-          "basic"
-        ]
+        "path": "hub/sdk_scheme_adapter/bulk/basic/bulk-happy-path.json"
```

Comment:
Why labels are removed?

Reply:
@vijayg10 The labels are removed by TTK on export.